docs/user/dashboard/aggregation-based.asciidoc

```diff
@@ -176,6 +176,48 @@ image:images/bar-chart-tutorial-2.png[Bar chart with sample logs data]
 . Click *Save and return*.
 
 
+[float]
+[[new-charts-library]]
+==== Check out our new charts library
+
+*Line, Area, Horizontal and Vertical Bar*
+
+From 7.12, XY axis charts use a new charts library. 
+
+NOTE: The legacy XY charts library will be removed in 7.16. If you are still using it, turn off the *XY axis legacy charts library* switch.
```

Comment:
This note seems like the kind of thing that belongs at the very top of the page. Also, a wording suggestion:

```suggestion
NOTE: The legacy XY charts library which powers bar, area, and line charts will be removed in 7.16. If you are still using it, turn off the *XY axis legacy charts library* switch in *Advanced settings*.
```